<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AllUseClassClass</class>
 <widget class="QMainWindow" name="AllUseClassClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1248</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AllUseClass</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_originalImage">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>原图像</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_showOriginalImage">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>461</width>
      <height>461</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_processImages">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>50</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>处理图像</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_showProcessImage">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>90</y>
      <width>461</width>
      <height>461</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="zpushButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Adobe Arabic&quot;;
text-decoration: underline;</string>
    </property>
    <property name="text">
     <string>撤销</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>560</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Adobe Arabic&quot;;
text-decoration: underline;</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1248</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>预处理</string>
    </property>
    <addaction name="phqzaction"/>
    <addaction name="hdlsaction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>分割图像</string>
    </property>
    <addaction name="OTSUaction"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>形态学处理</string>
    </property>
    <addaction name="pzaction"/>
    <addaction name="fsaction"/>
    <addaction name="bgaction"/>
    <addaction name="kqaction"/>
    <addaction name="dmaction"/>
    <addaction name="hmaction"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>杂项</string>
    </property>
    <addaction name="wtaction"/>
    <addaction name="zcaction"/>
    <addaction name="yxdaction"/>
    <addaction name="jxdaction"/>
    <addaction name="zxaction"/>
    <addaction name="monoaction"/>
    <addaction name="maxAreaaction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="phqzaction">
   <property name="text">
    <string>平滑去噪</string>
   </property>
  </action>
  <action name="hdlsaction">
   <property name="text">
    <string>均值化</string>
   </property>
  </action>
  <action name="OTSUaction">
   <property name="text">
    <string>OTSU</string>
   </property>
  </action>
  <action name="wtaction">
   <property name="text">
    <string>检测圆</string>
   </property>
  </action>
  <action name="zcaction">
   <property name="text">
    <string>周长</string>
   </property>
  </action>
  <action name="yxdaction">
   <property name="text">
    <string>圆形度</string>
   </property>
  </action>
  <action name="jxdaction">
   <property name="text">
    <string>矩形度</string>
   </property>
  </action>
  <action name="zxaction">
   <property name="text">
    <string>重心</string>
   </property>
  </action>
  <action name="pzaction">
   <property name="text">
    <string>膨胀</string>
   </property>
  </action>
  <action name="fsaction">
   <property name="text">
    <string>腐蚀</string>
   </property>
  </action>
  <action name="bgaction">
   <property name="text">
    <string>闭关</string>
   </property>
  </action>
  <action name="kqaction">
   <property name="text">
    <string>开启</string>
   </property>
  </action>
  <action name="dmaction">
   <property name="text">
    <string>顶帽</string>
   </property>
  </action>
  <action name="hmaction">
   <property name="text">
    <string>黑帽</string>
   </property>
  </action>
  <action name="monoaction">
   <property name="text">
    <string>几个物体</string>
   </property>
  </action>
  <action name="maxAreaaction">
   <property name="text">
    <string>最大物体面积</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
